loader-tests/README.md

```diff
@@ -0,0 +1,3 @@
+# Slow Reader Loader tests
+
+Loader testing tool for OPML files.
```

Comment:
We need a little more details here. At least longer description as we have in `README.md` and explanation why we need this dir (to test that we can load posts from real Internet, not abstract unit tests).

Also, we will need a guide how to run OMPL test.

Reply:
Sure - it's a draft PR, I actually have some questions but you are too fast :grin: